apps/taxes/graphql/fragments/TaxBase.graphql

```diff
@@ -60,11 +60,16 @@ fragment TaxBase on TaxableObject {
     ...TaxBaseLine
   }
   sourceObject {
+    __typename
     ... on Checkout {
       avataxEntityCode: metafield(key: "avataxEntityCode")
+      email
     }
     ... on Order {
       avataxEntityCode: metafield(key: "avataxEntityCode")
+      user {
```

Comment:
**Question[^1][^2]**:
Orders for anonymous users keep email in separate field - `userEmail`. Should it be used in your case too? 

[^1]: ✅ Not blocking the review. 
[^2]: I just wanted to know more context, or spotted a cool solution I would like to know more about :)

Reply:
good catch, fixed it